onnxruntime/python/tools/quantization/operators/where.py

```diff
@@ -0,0 +1,56 @@
+import onnx
+
+from ..quant_utils import TENSOR_NAME_QUANT_SUFFIX, QuantizedValue, QuantizedValueType, attribute_to_kwarg
+from .base_operator import QuantOperatorBase
+from .qdq_base_operator import QDQOperatorBase
+
+
+class QWhere(QuantOperatorBase):
+    def quantize(self):
+
+        node = self.node
+        (
+            quantized_input_names,
+            zero_point_name,
+            scale_name,
+            nodes,
+        ) = self.quantizer.quantize_inputs(node, [1, 2])
+        if quantized_input_names is None:
+            return super().quantize()
+        quantized_output_name = node.output[0] + TENSOR_NAME_QUANT_SUFFIX
+        q_output = QuantizedValue(
+            node.output[0],
+            quantized_output_name,
+            scale_name,
+            zero_point_name,
+            QuantizedValueType.Input,
+        )
+        self.quantizer.quantized_value_map[node.output[0]] = q_output
+        quantized_node_name = ""
+        if node.name != "":
+            quantized_node_name = node.name + "_quant"
+
+        kwargs = {}
+        for attribute in node.attribute:
+            kwargs.update(attribute_to_kwarg(attribute))
+
+        quantized_input_names = quantized_input_names.extend(node.input[1:])
```

Comment:
you need to add input[0]. This will add input[1] and input[2]?

Reply:
The input[0] is Boolean for conditions, do we need to quantized bool?